Intersect.Editor/Forms/Editors/frmSpell.cs

```diff
@@ -216,7 +216,7 @@ private void InitLocalization()
             grpHotDot.Text = Strings.SpellEditor.hotdot;
             chkHOTDOT.Text = Strings.SpellEditor.ishotdot;
             lblTick.Text = Strings.SpellEditor.hotdottick;
-            lblOTanimationDisclaimer.Text = "(Set to " + Strings.General.none + " to keep same animation)";
+            lblOTanimationDisclaimer.Text = Strings.SpellEditor.overTimeDisclaimer1 + Strings.General.none + Strings.SpellEditor.overTimeDisclaimer2;
```

Comment:
Yup, you did. Look at other strings, there's a parameterization feature.

Reply:
Good stuff, thank you. I like that